src/mount/zwo.rs

```diff
@@ -164,7 +164,7 @@ impl Mount for ZWO {
 
         send_cmd_and_get_reply(
             &mut self.serial_port,
-            format!(":Rg{:.2}#", a1_s.max(0.1)),
+            format!(":Rg{:.2}#", a1_s.max(0.9)),
```

Comment:
@ddompe The 0.1 was here by mistake, wasn't it? It should indeed be 0.9 (max guiding rate supported by ZWO).

Reply:
Right, copy & paste typo on my side